packages/fe-fpm-writer/src/action/types.ts

```diff
@@ -0,0 +1,28 @@
+import { CustomElement, InternalCustomElement, Position } from '../common/types';
+
+export enum ControlType {
+    header = 'header',
+    footer = 'footer',
+    facet = '@com.sap.vocabularies.UI.v1.FieldGroup',
```

Comment:
FieldGroup refers more to a form than a facet, have a look at https://ui5.sap.com/#/topic/7619517a92414e27b71f02094bd08d06 for details on the different action extension that are supported